layouts/partials/dashboards.html

```diff
@@ -167,11 +167,14 @@ <h6><a href="{{ .Page.RelPermalink }}">{{ .Name }}</a></h6>
                  month: 'short',
                  day: 'numeric'
              });
-
              cardData.push({ card, updatedDate });
 
          } else {
-             spanElement.textContent = 'No matching entry in EBI index';
+             // In case there is no entry in the EBI file yet,
+             // we add a mock date for the reordering to keep working
+             spanElement.textContent = 'no matching entry in EBI index';
```

Comment:
Maybe this can be just "Not available", we developers would know what it means and also might be more meaningful to general public ?